rpcs3/CMakeLists.txt

```diff
@@ -118,6 +118,7 @@ ${LLVM_INCLUDE_DIRS}
 "${RPCS3_SRC_DIR}/../rsx_program_decompiler/shader_code"
 "${RPCS3_SRC_DIR}/../Vulkan/Vulkan-LoaderAndValidationLayers/include"
 "${RPCS3_SRC_DIR}/../Vulkan/glslang/glslang/Public"
+${ZLIB_INCLUDE_DIRS}
```

Comment:
It should be included before RPCS3 itself, like all the other external/3rdparty things.